README.md

```diff
@@ -152,7 +165,10 @@ flake8 .
 ### Collaboration
 You need to have PostgresQL installed and set up on your machine.
 
-Clone the repository and please read the [contributing guide](/CONTRIBUTING.md).
+Clone the repository from the `staging` branch and please read the [contributing guide](/CONTRIBUTING.md).
+
+You may also need to have `Heroku`, `Ruby` and `Travis CI` installed and signed in. 
```

Comment:
_Travis_ and Heroku command line clients_.

Also, before Travis can be installed, it's in their official documentation that certain Ruby versions need to be installed. Since the complete instructions for Ruby installation is also there, I don't think there's any point stating it here, they'll see it in the Travis documentation. Maybe remove Ruby, and also add that the Heroku and Travis login commands needs to be run in the terminal after the command line clients have been successfully installed.